deployment/environment/devenv/don.go

```diff
@@ -381,3 +419,15 @@ func (n *Node) ReplayLogs(blockByChain map[uint64]uint64) error {
 	}
 	return nil
 }
+
+func ptr[T any](v T) *T {
+	return &v
+}
+
+func value[T any](v *T) T {
```

Comment:
It seems important for this helpers behavior to be clear at call sites:
```suggestion
func valOrZero[T any](v *T) T {
```
```suggestion
func orZero[T any](v *T) T {
```
:shrug:

Reply:
sure. can change in followup?

ci is brutal and need to incorporate down streams 🙏